src/app/icd/IcdManagementServer.h

```diff
@@ -30,21 +30,11 @@ using chip::Protocols::InteractionModel::Status;
 class IcdManagementServer
 {
 public:
-    void Init(uint32_t idle_interval, uint32_t active_interval, uint16_t active_threshold, uint32_t count,
-              uint16_t fabric_clients_supported)
-    {
-        mIdleInterval           = idle_interval;
-        mActiveInterval         = active_interval;
-        mActiveThreshold        = active_threshold;
-        mIcdCounter             = count;
-        mFabricClientsSupported = fabric_clients_supported;
-    }
+    uint32_t GetIdleModeInterval() { return mIdleInterval_s; }
 
-    uint32_t GetIdleModeInterval() { return mIdleInterval; }
+    uint32_t GetActiveModeInterval() { return mActiveInterval_ms; }
```

Comment:
Technically methods should have unit as well (`GetActiveModeIntervalMs`).

Reply:
done